src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -708,5 +799,38 @@ object EntityClient {
   enum GenerationType:
     case Original, Derived
 
-  case class EntityMetadata(entityNode: Node, identifiers: Seq[Node], metadataNodes: Seq[Elem])
+  sealed trait EntityMetadata:
+    val entityNode: Node
+    val identifiers: Seq[Node]
+    val links: Seq[Node]
+    val metadataNodes: Seq[Node]
+    val eventActions: Seq[Node]
+
+  case class StandardEntityMetadata(
```

Comment:
What does `StandardEntityMetadata` represent? If it's not an IO or a CO then should this be `SoMetadata`?

Reply:
`StandardEntityMetadata` is the non-specific metadata that is common for all Entity types. The reason why I didn't call it `SoMetadata`, despite that being the only option is:

1. it doesn't contain SO-specific metadata (so I'm just making it clear with this name)
2. In case another Entity type is created in the future. This is unlikely but there are cases in the Preservica client where we've assumed that there could be a new type in the future, like `fromType`, so I just used that as an example

Reply:
Ok cool. Could you put a comment above it just to say that briefly.